dataprovider/providers/cloud/data_provider.go

```diff
@@ -32,12 +34,17 @@ const (
 	cloudAccountNameField = "cloud.account.name"
 	cloudProviderField    = "cloud.provider"
 	cloudRegionField      = "cloud.region"
+	// TODO: update fields names when an ECS field is decided
```

Comment:
can we track it somehow? maybe add a link to this part in the ticket about the ECS?

Reply:
Is there such a ticket?